### PR TITLE
Update definition in Chinese DEFINITION.md

### PR DESCRIPTION
**Pull request by towonzhou**
更清楚的表达了本质,把示例明细与定义表述分开

### CODE REVIEW

**Comment by towonzhou** [comment]
update definition in Chinese

**Review by tao12345666333 — COMMENTED** [review]
Personal opinion, I don't think this modification makes sense

File: DEFINITION.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ innovations accessible for everyone.
 
 ## 中文版本：
 
-云原生技术有利于各组织在公有云、私有云和混合云等新型动态环境中，构建和运行可弹性扩展的应用。云原生的代表技术包括容器、服务网格、微服务、不可变基础设施和声明式API。
+云原生技术有利于各组织在新型动态环境中构建和运行可弹性扩展的应用，动态环境包括共有云,私有云和混合云等。云原生的代表技术包括容器、服务网格、微服务、不可变基础设施和声明式API。
```
Comment (nobodyiam) [inline DEFINITION.md:35]:
```suggestion
云原生技术有利于各组织在新型动态环境中构建和运行可弹性扩展的应用，动态环境包括共有云，私有云和混合云等。云原生的代表技术包括容器、服务网格、微服务、不可变基础设施和声明式API。
```

**Comment by resouer** [comment]
I tried to digest the PR for a few times but still didn't feel the proposed change is significantly better than original narrative ^^ 

I will close it, really appreciated though!

**Pull request closed without merging**